enos/enos-scenario-openldap.hcl

```diff
@@ -508,9 +508,9 @@ scenario "openldap" {
 
 
       plugin_name         = "vault-plugin-secrets-openldap"
-      plugin_dest_dir     = "/Users/USER/go/vault-plugins"
+      plugin_dest_dir     = "/Users/hamzaelmokhtarshili/go/vault-plugins"
```

Comment:
Should we add a comment above all these to indicate that they are placeholder values?